src/assets/roadmap/roadmap.json

```diff
@@ -46,64 +46,64 @@
         "fr": "Mars 2022 - Avril 2022"
       },
       "description": {
-        "en": "Integration of stakepool metadata, and enhanced metadata and other offchain sources. Integration of asset metadata, and separation of fungible and non fungible assets. Integration of additional wallet types, legacy byron, icarus and non-staking shelley.",
-        "fr": "Intégration des métadonnées du pool de participations et autres sources hors chaîne. Intégration des métadonnées des atouts et séparation des atouts fongibles et non fongibles. Intégration de types de portefeuilles supplémentaires, legacy byron, icarus et non-staking shelley."
+        "en": "Integration of stakepool metadata, and enhanced metadata and other offchain sources.",
+        "fr": "Intégration des métadonnées du pool de participations et autres sources hors chaîne."
       }
     },
     {
       "title": {
-        "en": "Beta Updates",
+        "en": "Backend Rewrite",
         "fr": "Mises à jour bêta"
       },
       "timeline": {
-        "en": "April 2022 - May 2022",
-        "fr": "Avril 2022 - Mai 2022"
+        "en": "April 2022 - December 2022",
+        "fr": "Avril 2022 - Octobre 2022"
       },
       "description": {
-        "en": "Integration of an Asset Gallery for displaying NFT and other assets associated with a wallet. Ability to display stakepool overviews and search for stakepool data and operators.",
-        "fr": "Intégration d'une galerie d'atouts pour afficher des NFT et autres associés à un portefeuille. Afficher des aperçus de pools et de recherche des données et des opérateurs de pools."
+        "en": "Developing updated API's using CRQS/ES API management to improve performance of larger wallets and eliminate the need for realtime queries.",
+        "fr": "Développer des API mises à jour à l'aide de la gestion des API CRQS/ES pour améliorer les performances des portefeuilles plus volumineux et éliminer le besoin de requêtes en temps réel."
       }
     },
     {
       "title": {
-        "en": "Live Release 1",
-        "fr": "Version 1"
+        "en": "Preview and Preprod",
+        "fr": "Preview et Preprod"
       },
       "timeline": {
-        "en": "Spring 2022",
-        "fr": "Printemps 2022"
+        "en": "October 2022 - December 2022",
+        "fr": "Octobre 2022 - Decembre 2022"
       },
       "description": {
-        "en": "Migration to live servers and www domain instead of the beta domain. Verification of language packs.",
-        "fr": "Migration vers des serveurs de production et un domaine www au lieu du domaine bêta. Vérification des traductions."
+        "en": "Developing updated API's using CRQS/ES API management to improve performance of larger wallets and eliminate the need for realtime queries.",
+        "fr": "Développer des API mises à jour à l'aide de la gestion des API CRQS/ES pour améliorer les performances des portefeuilles plus volumineux et éliminer le besoin de requêtes en temps réel."
```

Comment:
Mises à jour des APIS avec l'aide de CRQS/ES pour améliorer le performance des portefeuilles volumineux et éliminer le besoin de requêtes en temps réel.

Reply:
just noticed, this was completely wrong text.  Should be:

"Added support for new preprod and preview networks"

+ 

French

Reply:
I thought it was weird to see the same text twice 🤷

Reply:
What's the french translation for that?

Reply:
Ajout des nouveaux réseaux de préprod / preview